client/goclient.go

```diff
@@ -46,6 +46,11 @@ type Client struct {
 	smCrypto          bool
 }
 
+const (
+	V2_1_0 = "2.1.0"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported const V2_1_0 should have comment (or a comment on this block) or be unexported](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=4970769)